include/bluetooth/services/fast_pair.h

```diff
@@ -77,13 +71,41 @@ enum bt_fast_pair_battery_comp {
 	BT_FAST_PAIR_BATTERY_COMP_COUNT
 };
 
+/** @brief Fast Pair discoverable advertising info. Currently empty. */
+struct bt_fast_pair_disc_adv_info {};
+
+/** @brief Fast Pair not discoverable advertising info. */
+struct bt_fast_pair_not_disc_adv_info {
+	/** Show advertising UI indication flag. According to Fast Pair specification,
+	 *  when no Account Key has been saved on the device, this flag has no effect on
+	 *  the advertising data.
+	 */
+	bool show_adv_ui_indication;
+
+	/** Fast Pair advertising battery mode. */
+	enum bt_fast_pair_adv_battery_mode adv_battery_mode;
+};
+
+/** @brief Fast Pair advertising info. Content depends on Fast Pair advertising mode
+ *  (see @ref bt_fast_pair_adv_mode and @ref bt_fast_pair_adv_config).
+ */
+union bt_fast_pair_adv_info {
+	/** Fast Pair discoverable advertising info. */
+	struct bt_fast_pair_disc_adv_info discoverable;
```

Comment:
I think we should be consistent if the struct type uses `disc` infix we should also use the similar form to name fields